kubeapps-dashboard.jsonnet

```diff
@@ -130,6 +138,18 @@ local serviceDeployFromValues(parentName, componentName, values) = {
                   config: {mountPath: "/monocular/config"},
                 },
               },
+              local tillerContainer = (import "tiller-deployment.jsonnet").spec.template.spec.containers[0],
+              tiller: tillerContainer {
+                overrideEnvs(overrides):: [
```

Comment:
wasn't sure if there was a better way to do this @anguslees, I tried to use `kube.Container` since it already has `_env` but it didn't quite work correctly because the tiller-deploy is pure json. I'm trying to keep this as compatible as the pure json as possible to make it easier to upgrade tiller-deployment later.

Reply:
Right, this sort of use-case is the reason I convert as much as possible to named dictionaries in kube.libsonnet (`containers_`, `env_`, `args_`, etc).

Your `overrideEnvs` function is a great solution for working with the existing array.  :+1: